content/30.abstracting_simulation_types.md

```diff
@@ -283,3 +283,10 @@ Despite these weak spots, however, `yt` does provide a number of routines that a
 (See @sec:pixelization for more detail on this process.)
 Pixelizers that take variable-resolution data along the $r$ and $\theta$ axes have been made available (for slicing along a conical section of a sphere or along the $z$ axis of a cylinder) as well as very simple projections from the surface of a sphere to a flat image (specifically utilizing the Aitoff projection).
 `yt` also provides access to Cartopy [@doi:10.5281/zenodo.7430317] for more advanced or featureful map projections, as well as overlaying continents and other geographic shapes.
+In Figure @fig:spherical_data we demonstrate some of the native, built-in functionality `yt` provides for non-cartesian data.
+This dataset, a simulation of MHD-driven winds in a protoplanetary disk, was conducted in spherical coordinates.
+Here, we have used `yt`'s functionality for overplotting streamlines as well as line integral convolution on irregular meshes to display the data in its native resolution and as a slice along the poloidal axis of the simulation domain.
```

Comment:
"in the poloidal plane" or "along the azimuth axis" are equally valid, but "along the poloidal axis" isn't
```suggestion
Here, we have used `yt`'s functionality for overplotting streamlines as well as line integral convolution on irregular meshes to display the data in its native resolution and as a slice along the azimuthal axis of the simulation domain.
```